.travis.yml

```diff
@@ -26,7 +27,7 @@ install:
 script:
   - mkdir -p build/logs
   - "if [[ $RUN_LINTER ]]; then vendor/bin/php-cs-fixer fix --verbose --diff --dry-run --config=.php_cs; fi"
-  - "if [[ $RUN_LINTER ]]; then vendor/bin/phpcs --standard=PSR2 src/; fi"
+  - "if [[ $RUN_LINTER ]]; then vendor/bin/phpcs -n --standard=PSR2 src/; fi"
```

Comment:
Again, we want the warnings and we don't want the whitespace changes to make very long lines.

Reply:
Hi @oschwald, thanks for your reply.

If we set the `-n` to ignore warning message, the `PHP_CodeSniffer` will be passed.

And the `PHP-CS-Fixer` will be also passed.

We remove `-n` from `PHP_CodeSniifer` coding style check and fix them.

Then the `PHP-CS-Fixer` will have the problematic coding style.

They're conflicted. The related Travis CI build log is as follows:

- The problematic coding style from `PHP-CS-Fixer`.
https://travis-ci.org/open-source-contributions/web-service-common-php/jobs/607750057?utm_medium=notification&utm_source=github_status

Reply:
Aha, I apologize; I did not realize this was a [new rule in php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4452). It was only released a few days ago.

I think we could just disable that rule by adding `'single_line_throw' => false,` to `.php_cs`.

Reply:
I disabled that rule on all of the repos. If you rebase, you should get it.